userspace/libsinsp/sinsp.cpp

```diff
@@ -1870,6 +1870,16 @@ void sinsp::get_capture_stats(scap_stats* stats) const
 	}
 }
 
+struct scap_stats_v2* sinsp::get_capture_stats_v2(uint32_t flags, uint32_t* nstats, int32_t* rc) const
+{
+	scap_stats_v2* stats_v2 = scap_get_stats_v2(m_h, flags, nstats, rc);
+	if(*rc != SCAP_SUCCESS)
+	{
+		throw sinsp_exception(scap_getlasterr(m_h));
+	}
+	return stats_v2;
```

Comment:
I'm not sure we want to kill the whole process if we are not able to collect stats, WDYT? Maybe the consumer should just check the `rc`  and the return value `!=NULL` (?) :thinking: 

```suggestion
	return scap_get_stats_v2(m_h, flags, nstats, rc);	
```